windows/test-app.js

```diff
@@ -190,21 +165,22 @@ function copyProjectTemplateAndReplace(
     for (const resource of resourcesPaths) {
       copyAndReplaceAll(
         path.relative(path.dirname(manifestFilePath), resource),
-        destPath,
-        path.join(windowsDir, projDir, "Bundle", path.basename(resource))
+        projectFilesDestPath,
+          path.join(bundleDir, path.basename(resource))
       );
```

Comment:
Can we avoid copying assets to `node_modules/.generated/windows/ReactTestApp/Bundle`, and instead create reference to them in `.vcxproj`? I think it will be confusing to users if they have to run `test-app.js` every time they update any assets.

Reply:
Sure, we could do that. Users would still need to rerun the script each time they change resources paths in manifest, but that will probably not happen so frequently.

Reply:
Yeah, we can only add paths to resources we know of. 😄